unique_toolkit/unique_toolkit/content/functions.py

```diff
@@ -204,6 +204,49 @@ def _upsert_content(
     )  # type: ignore
 
 
+def upload_content_from_bytes(
+    user_id: str,
+    company_id: str,
+    content: bytes,
+    content_name: str,
+    mime_type: str,
+    scope_id: str | None = None,
+    chat_id: str | None = None,
+    skip_ingestion: bool = False,
+):
+    """
+    Uploads content to the knowledge base.
+
+    Args:
+        user_id (str): The user ID.
+        company_id (str): The company ID.
+        content (bytes): The content to upload.
+        content_name (str): The name of the content.
+        mime_type (str): The MIME type of the content.
+        scope_id (str | None): The scope ID. Defaults to None.
+        chat_id (str | None): The chat ID. Defaults to None.
+        skip_ingestion (bool): Whether to skip ingestion. Defaults to False.
+
+    Returns:
+        Content: The uploaded content.
+    """
+
+    try:
+        return _trigger_upload_content(
+            user_id=user_id,
+            company_id=company_id,
+            content=content,
+            content_name=content_name,
+            mime_type=mime_type,
+            scope_id=scope_id,
+            chat_id=chat_id,
+            skip_ingestion=skip_ingestion,
+        )
+    except Exception as e:
```

Comment:
Please lets be specific about th error being raised, else we don't need to put this in try except, it would show anyway in the stack-trace and we wouldn't need any logger.